environments/sso_vars.yaml

```diff
@@ -0,0 +1,30 @@
+# Parameters for single-signon.yaml and enable-federation-openidc.yaml
+parameter_defaults:
+  KeystoneTrustedDashboards: https://esi.massopen.cloud/
+  # Copy the IdP name from moc openstack. Should we use a different provider for ESI?
+  KeystoneOpenIdcIdpName: moc
+  # Copy the ProviderMetadataUrl from moc openstack. Same as above.
+  KeystoneOpenIdcProviderMetadataUrl: https://sso.massopen.cloud/auth/realms/moc/.well-known/openid-configuration
+  # I guess OpenID client ID should be the identifier for ESI deployment. So we need a new client ID for it.
+  KeystoneOpenIdcClientId: esi
```

Comment:
You would need to request to have a client created in the MOC sso environment; they would provide you with the client id (you could request `esi`) and other necessary credentials, like the client secret.

Reply:
Done. Naved created a client for esi.